packages/minter-contracts/ligo/src/swaps/fa2_swap.mligo

```diff
@@ -36,15 +36,23 @@ type swap_entrypoints =
 type swap_storage =
   { next_swap_id : swap_id
   ; swaps : (swap_id, swap_info) big_map
+#if BURN_PAYMENT
+  ; burn_address : address
+#endif
   }
 
 type return = operation list * swap_storage
 
 (* ==== Values ==== *)
 
+[@inline] let null_address = ("tz1Ke2h7sDdakHJQh8WX4Z372du1KChsksyU" : address)
```

Comment:
The null address is the hash of 0x000 public key and is used in the liquidity baking contract. I don't really understand however why its theoretically possible that it is not claimed however. `burn_address` is still configurable, this is just used as an example storage.

Reply:
Ok, great that it's still configurable. Maybe then rename to `example_burn_address` or otherwise self-document a bit?

It's _possible_ that it's been claimed: there's multiple choices for a "zero address" so someone could find an address that "looked null" but isn't. Using a contract that we control, e.g. `parameter unit; storage unit; code { FAILWITH }`, gives us better guarantees and makes accounting easier.